src/config/configOptionMigrations.js

```diff
@@ -23,6 +24,26 @@ module.exports = {
     // Set DashPay contract ID and block height for testnet
     lodashSet(options, 'platform.dashpay', systemConfigs.testnet.platform.dashpay);
 
+    return options;
+  },
+  '0.17.4': (name, options) => {
+    // Skip if it's not a system config
+    if (!systemConfigs[name]) {
+      return options;
+    }
+
+    const previousStdoutLogLevel = lodashGet(
+      options,
+      'platform.drive.abci.log.level',
+      systemConfigs[name].platform.drive.abci.log,
+    );
+
+    // Set Drive's new logging variables
+    lodashSet(options, 'platform.drive.abci.log', systemConfigs[name].platform.drive.abci.log);
+
+    // Keep previous log level for stdout
+    lodashSet(options, 'platform.drive.abci.log.stdout.level', previousStdoutLogLevel);
```

Comment:
You need to delete `platform.drive.abci.log.level` - upgrading from 0.17.3 returns this error for me:
```
InvalidConfigFileFormatError: Invalid '/home/strophy/.mn/config.json' config format: config.platform.drive.abci.log should NOT have additional properties
```

Reply:
Thank you. Fixed.

Reply:
I'm still getting the same error. The `platform.drive.abci.log.level` setting cannot exist after the migration is complete, it must be manually deleted like `lodashUnset(options, 'baseConfig.platform.drive.abci.log.level')` or something similar.

Reply:
I do not understand why it doesn't work for you. `baseConfig.platform.drive.abci.log.level` is not present. System configs are updated and don't contain `log.level` anymore. Could you please attach your config file so I will be able to debug this problem?

Reply:
@shumkov @strophy The migration wont run until we release `0.17.4` version of `mn-bootstrap` hence the error.

Reply:
Yeah, that's expected behaviour.

Reply:
Thanks, that explains why it wasn't working when I tested.